### PR TITLE
Remove OT info from Content Index article.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

**Comment by jpmedley** [comment]
@jeffposnick Even if you approve the changes, I can't merge this until I make sure the interface images are up to date.

File: src/site/content/en/blog/content-indexing-api/index.md [hunk 1]
```diff
@@ -5,28 +5,19 @@ authors:
   - jeffposnick
 description: Your PWA might cache articles and media files, but how will your users know that your pages work while offline? The Content Indexing API is one answer to this question currently in an origin trial. Once the index is populated with content from your PWA, as well as any other installed PWAs, it will show up in dedicated areas of supported browsers.
 date: 2019-12-12
-updated: 2020-05-07
+updated: 2020-05-26
```
Comment (jeffposnick) [inline src/site/content/en/blog/content-indexing-api/index.md:8]: Just double-checking that you intended May 26. I assume that's timed for M84.

Comment (jpmedley) [reply src/site/content/en/blog/content-indexing-api/index.md:8]: The only other real choice is the 27th. I see no reason to wait. I'd be fine with pushing it today if it were ready.

**Comment by kaycebasques** [comment]
@jpmedley please update the image markup to use the [partial screenshot markup](https://web.dev/handbook/markup-media/#partial-screenshots) or [entire screenshot markup](https://web.dev/handbook/markup-media/#entire-screenshots) when you update the images (and remember to run the updated images through tinypng.com or something similar)

File: src/site/content/en/blog/content-indexing-api/index.md (hunk 1 above)
Comment (jpmedley) [reply src/site/content/en/blog/content-indexing-api/index.md:8]: M84 is beta on Thursday. I intended this to go out this week. I'm linking to it from the beta release post.

**Comment by jpmedley** [comment]
@kaycebasques Done.

**Pull request merged**